lib/api/src/native.rs

```diff
@@ -112,8 +112,26 @@ macro_rules! impl_native_traits {
             $( $x: FromToNativeWasmType, )*
             Rets: WasmTypeList,
         {
+
             /// Call the typed func and return results.
             pub fn call(&self, $( $x: $x, )* ) -> Result<Rets, RuntimeError> {
+                self.call_(true, $( $x, )*)
+            }
+
+            /// Call the typed func and return results.
+            /// This would not catch any traps and must be called within a callback passed to `wasmer_vm::catch_traps`.
+            pub fn call_unchecked(&self, $( $x: $x, )* ) -> Result<Rets, RuntimeError> {
```

Comment:
This function must be `unsafe` with prerequisites listed in a `# Safety` section of a doc comment.